scene/main/scene_tree.cpp

```diff
@@ -620,6 +620,12 @@ void SceneTree::_notification(int p_notification) {
 
 			get_root()->propagate_notification(p_notification);
 		} break;
+		case NOTIFICATION_WM_MOUSE_ENTER: {
```

Comment:
These cases should probably be added to the group in line 619 to make it look like this:
```c++
                case NOTIFICATION_OS_MEMORY_WARNING:
                case NOTIFICATION_WM_FOCUS_IN:
                case NOTIFICATION_WM_FOCUS_OUT: 
                case NOTIFICATION_WM_MOUSE_ENTER: 
                case NOTIFICATION_WM_MOUSE_EXIT: {

                        get_root()->propagate_notification(p_notification);
                } break;
```
I'm not super familiar with this code but if these should be propagated like that there then that is probably how to do it!

Reply:
Oh well... I´m not familiar at all with c++ syntax :D :D :D :D....I try the fix and works but the two bots report error so i fix that.